### DISCUSSION TITLE
Integration

### DISCUSSION THREAD

**Discussion by ecalfiqueo** [BanDev/Notify#91] [General]
How can I add it to discord?

**Comment by russellbanks** [comment]
1. Download `Notify-[version]-all.jar` from releases.
2. Create a `.env` file in the same directory as the jar file.
3. Add your API key to the `.env` file:

    ```env
    DISCORD_API_KEY=[YOUR_API_KEY_HERE]
    ```
5. Run the JAR
    ```pwsh
    java -jar Notify-[version]-all.jar
    ```
